packages/portal/spec-renderer-mini/src/components/OperationsListItem.vue

```diff
@@ -0,0 +1,171 @@
+<template>
+  <button
+    class="item"
+    :class="{ 'item--selected': isSelected, [`item--method-${item.method}`]: true }"
+    :data-testid="`mini-spec-item-${item.path}`"
+    type="button"
+    @click="emit('click', item)"
+  >
+    <h2
+      class="summary truncate"
+      :title="item.summary ?? ''"
+    >
+      {{ item.summary }}
+    </h2>
```

Comment:
question: should there be a `v-if` here to hide the `h2` tag if `summary` is empty?